### PR TITLE
fix: Handle fragment transaction correctly on rotation

### LINKED ISSUE #738: Rotating screen anywhere opens Tickets fragment
**Describe the bug**

Rotating the screen on any fragment opens up the Tickets fragment, even though the selected item on the navigation bar remains the same and does not change to Tickets.

**To Reproduce**

1. Go to any fragment.
2. Turn on screen rotation if it is off.
3. Rotate the mobile.
4. App will switch to Tickets fragment.

**Expected behavior**
The fragment should not have changed. 

**GIF**

ezgif com-video-to-gif 4

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #738 Rotating screen anywhere opens Tickets fragment 

**Changes:**
 
Now the fragment would not change to Tickets fragment when the screen is rotated.

**GIF for the change:**

ezgif com-video-to-gif 5

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,7 +16,8 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".MainActivity"
-            android:windowSoftInputMode="adjustPan">
+            android:windowSoftInputMode="adjustPan"
+            android:configChanges="orientation">
```
Comment (nikit19) [inline app/src/main/AndroidManifest.xml]: Rather, why don't you try to eliminate the root cause

Comment (ShridharGoel) [reply app/src/main/AndroidManifest.xml]: Shouldn't we use this so that activity is not recreated on rotating the screen?

Comment (iamareebjamal) [reply app/src/main/AndroidManifest.xml]: No, what if we use different layouts for portrait and landscape. This is a workaround, not a fix

Comment (ShridharGoel) [reply app/src/main/AndroidManifest.xml]: Okay, I'll work on it.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -38,4 +38,4 @@
             android:value="${STRIPE_API_TOKEN}"/>
     </application>
 
-</manifest>
\ No newline at end of file
+</manifest>
```
Comment (iamareebjamal) [inline app/src/main/AndroidManifest.xml]: Revert changes in this file

Comment (ShridharGoel) [reply app/src/main/AndroidManifest.xml]: Done

**Comment by ShridharGoel** [comment]
@nikit19 @iamareebjamal Is this fine now?

**Comment by iamareebjamal** [comment]
Looks fine for me, let NIkit review it

**Comment by liveHarshit** [comment]
@iamareebjamal @nikit19 @simarsingh24 
This contains bugs because, in many places, Flag activity clear top is used. In that case, while intent to the main activity 
``` java
if (savedInstanceState == null) {
            loadFragment(supportFragmentManager, EventsFragment(), frameContainer.id)
        }

        else
```
Only if statement is executed and all arguments are not used.
Here I open an issue #744 which have the same region.
Should I change the condition or sometiong else for the issue? Please suggest.

**Pull request merged**